doc/src/geturl.rst

```diff
@@ -0,0 +1,82 @@
+.. index:: geturl
+
+geturl command
+==============
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   geturl url keyword args ...
+
+* url = URL of the file to download
+* zero or more keyword argument pairs may be provided
+* keyword = *output* or *verify* or *overwrite* or *verbose*
+
+  .. parsed-literal::
+
+     *output* filename = write to *filename* instead of inferring the name from the URL
+     *verify* yes/no = verify SSL certificate and hostname if *yes*, do not if *no*
+     *overwrite* yes/no = if *yes* overwrite the output file in case it exists, do not if *no*
+     *verbose* yes/no = if *yes* write verbose debug output from libcurl to screen, do not if *no*
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   geturl https://www.ctcms.nist.gov/potentials/Download/1990--Ackland-G-J-Vitek-V--Cu/2/Cu2.eam.fs
+   geturl https://github.com/lammps/lammps/blob/develop/bench/in.lj output in.bench-lj
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Download a file from an URL to the local disk. This is implemented with
+the `libcurl library <https:://curl.se/libcurl/>`_ which supports a
+large variety of protocols including "http", "https", "ftp", "scp",
+"sftp", "file".  The transfer will only be performed on MPI rank 0.
+
+The *output* keyword can be used to set the filename. By default, the last part
+of the URL is used.
+
+The *verify* keyword allows to turn on or off whether ``libcurl`` will validate
```

Comment:
... keyword sets whether libcurl ...

Reply:
... keyword determines whether libcurl ...

(more consistent with description of next 2 keywords)

Reply:
done